### PR TITLE
Added intros

### PR DESCRIPTION
**Pull request by deakgeek**
#78

### CODE REVIEW

File: content/efcore/getting-started/loading-related-data.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Loading Related Data  
+# Loading Related Data
+
+In this lesson, we will cover how to load non-primitive types from the database, how chain commands to include multiple layers of related data, and how to use explicit loading and aggregate functions.
```
Comment (rdahlman3) [inline content/efcore/getting-started/loading-related-data.md]: how **to** chain commands

**Review by rdahlman3 — CHANGES_REQUESTED** [review]

File: content/efcore/getting-started/loading-related-data.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-# Loading Related Data  
+# Loading Related Data
+
+In this lesson, we will cover how to load non-primitive types from the database, how to chain commands to include multiple layers of related data, and how to use explicit loading and aggregate functions.
```
Comment (helmoski) [inline content/efcore/getting-started/loading-related-data.md]: Chaining commands to include multiple layers of related data won't really mean anything to them. Could we change this to:

"In this lesson, we will learn how to load non-primitive types from the database using both eager loading and explicit loading, as well as how to use aggregate functions to perform calculations on our data without loading it from the database."

File: content/efcore/getting-started/manipulating-entities.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Manipulating Entities 
  
-Four operations that we often perform on a record in a database are create, read, update, and delete (commonly referred to as "CRUD" operations). We walked through how to read records from a database in the [Querying Data](querying.md) tutorial. Now let's learn how to perform the other CRUD operations which allow us to manipulate records in the database. 
+Four operations that we often perform on a record in a database are create, read, update, and delete (commonly referred to as "CRUD" operations). We walked through how to read records from a database in the [Querying Data](querying.md) tutorial. In this lesson, we'll learn how to perform the other CRUD operations which allow us to manipulate records in the database.
```
Comment (helmoski) [inline content/efcore/getting-started/manipulating-entities.md]: ~~"in the [Querying Data](querying.md) tutorial."~~ => "in the [Querying Data](querying.md) lesson."

**Review by helmoski — CHANGES_REQUESTED** [review]

File: content/efcore/getting-started/loading-related-data.md (hunk 2 above)
Comment (arynhuck) [reply content/efcore/getting-started/loading-related-data.md]: 🌮

File: content/efcore/getting-started/manipulating-entities.md (hunk 3 above)
Comment (arynhuck) [reply content/efcore/getting-started/manipulating-entities.md]: 🌮

**Comment by arynhuck** [comment]
:shipit: ? @helmoski @rdahlman3

**Review by helmoski — APPROVED** [review]

**Comment by arynhuck** [comment]
Psyche, I merged master in. Now :shipit: ?

**Review by rdahlman3 — APPROVED** [review]

**Pull request merged**